README.md

```diff
@@ -73,6 +73,19 @@ You just need to use `deepview memory` instead of `deepview time`.
 python3 -m deepview_profile memory entry_point.py --output my_output_file.sqlite
 ```
 
+To export various available analysis to json file, you may use `deepview analysis --all` command for exact entry point and output file.
```

Comment:
I would add a note saying that this is required to later view the analysis on the web viewer.

Reply:
Please, check updated README file